scripts/train_svhn_tinierssd.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd ..
```

Comment:
Please remove `cd ..`. The scripts are expected to run as `scripts/xxxx.sh`

Reply:
Done and commit will be amended